tests/sentry/api/helpers/test_group_index.py

```diff
@@ -122,6 +123,9 @@ def test_resolving_unresolved_group(self, send_robust: Mock) -> None:
         unresolved_group = self.create_group(status=GroupStatus.UNRESOLVED)
         add_group_to_inbox(unresolved_group, GroupInboxReason.NEW)
         assert unresolved_group.status == GroupStatus.UNRESOLVED
+        assert GroupOpenPeriod.objects.filter(
+            group=unresolved_group, date_ended__isnull=True
+        ).exists()
```

Comment:
Would it make sense to add an assertion after the group is resolved to check that the open period has a not null `date_ended`?

Reply:
I tend to avoid asserts in non-test code so I was logging errors/exceptions if we found an unexpected inconsistency instead. The best way to monitor this might be a redash query on resolved groups joined with their open periods as we roll it out

Reply:
Oh I just meant a few lines further down in this test

Reply:
oops i misread that 🤦‍♀️ added that ty!